tck/src/main/java/ee/jakarta/tck/data/standalone/entity/EntityTest.java

```diff
@@ -132,6 +146,54 @@ public void testFinalSliceOfUpTo5() {
         assertEquals(false, it.hasNext());
     }
 
+    @Assertion(id = "133",
+               strategy = "Request the first KeysetAwareSlice of 6 results, expecting to find all 6, " +
+                          "then request the next KeysetAwareSlice and the KeysetAwareSlice after that, " +
+                          "expecting to find all results.")
+    public void testFirstKeysetAwareSliceOf6AndNextSlices() {
+        Pageable first6 = Pageable.ofSize(6);
+        KeysetAwareSlice<NaturalNumber> slice;
+
+        try {
```

Comment:
Is there any equivalent to `[EnabledIfSystemProperty](https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/condition/EnabledIfSystemProperty.html#named())` on Archilian?

IMHO: It should be better to disable a test than work with an exception